corehq/apps/reports/standard/cases/data_sources.py

```diff
@@ -159,14 +159,14 @@ def _get_username(self, user_id):
     def parse_date(self, date_string):
         try:
             return iso_string_to_datetime(date_string)
-        except:
+        except Exception:
```

Comment:
any chance this could be a more specific exception

Reply:
I didn't add this line. I only added `Exception` since my linter was complaining. I can remove if you think just having the bare `except` is better?

Reply:
haha, i was hoping you would know the specific exception(s?) this line is trying to catch but not blocking. I definitely like `Exception` better than bare `except` although im not sure why